ibm_watson/speech_to_text_v1_adapter.py

```diff
@@ -47,7 +43,6 @@ def recognize_using_websocket(self,
                                   speaker_labels=None,
                                   http_proxy_host=None,
                                   http_proxy_port=None,
-                                  customization_id=None,
```

Comment:
why are we getting rid of this? i think its still in the service.

Reply:
I asked about this a while ago - even though its deprecated it is still supported in the service. We should keep it in

https://ibm-analytics.slack.com/archives/CKFME903C/p1566841743005000